libbpfgo.c

```diff
@@ -131,6 +131,22 @@ int cgo_bpf_prog_detach_cgroup_legacy(int prog_fd,   // eBPF program file descri
     return syscall(__NR_bpf, BPF_PROG_DETACH, &attr, sizeof(attr));
 }
 
+struct bpf_link *cgo_bpf_program__attach_uprobe_multi(struct bpf_program *prog,
```

Comment:
Do we want to maintain the libbpf naming convention or keep libbpfgo's one?
See, `cgo_bpf_program__attach_uprobe_multi` vs `cgo_bpf_program_attach_uprobe_multi`